lib/assets/javascripts/utils/expandCollapseAll.js

```diff
@@ -40,7 +40,7 @@ export default () => {
     $.each($(el).children('a[data-toggle-direction]'), (i, a) => {
       $(a).click((event) => {
         event.preventDefault();
-        $(`#${accordion} div.panel-collapse`).collapse(`${$(a).attr('data-toggle-direction')}`);
+        $(`#${accordion}`).children('div.panel-collapse').collapse(`${$(a).attr('data-toggle-direction')}`);
```

Comment:
This change works but I just noticed that we have unnecessary loops and const here. This change should work too:

```
$('a[data-toggle-direction]').on('click', (e) => {
    e.preventDefault();
    const target = $(e.target);
    const accordion = target.parent().attr('data-parent');
    $(`#${accordion}`).children('div.panel-collapse').collapse(`${target.attr('data-toggle-direction')}`);
  });
```